### DISCUSSION TITLE
【新组件】 hexo- 主题 方格背景、公告模块

### DISCUSSION THREAD

**Discussion by expoli** [tangly1024/NotionNext#2028] [Ideas]
**为什么提出这个新的特性改动**
发现一个很好看的博客主题 [hexo-theme-butterfly](https://github.com/jerryc127/hexo-theme-butterfly) 希望能够移植过来。
这里推荐一个修改后的版本，感觉后面的方格背景还是不错的
- https://www.sszsj.cc/  
- https://github.com/csfwff/csfwff.github.io

- 预览
![图片](https://user-images.githubusercontent.com/31023767/213358734-992bbadf-da32-42e5-a39a-7e2e06a2b6f6.png)

**Comment by tangly1024** [comment]
NotionNext-hexo主题 其实就是来自 butterfly 移植，只是有些样式并没有完全按原主题设置。

另外，您的意思是：仅是很需要方格的背景吗，又或是需要移植此样式中的一些其他模块等。

**Comment by expoli** [comment]
是的、感觉方格背景，以及左右穿插的文章展示风格，还有对应的公告模块，也是挺不错的
